transient-extras-a2ps.el

```diff
@@ -2,9 +2,9 @@
 
 ;; Author: Samuel W. Flint <swflint@flintfam.org>
 ;; URL: https://git.sr.ht/~swflint/transient-extras-a2ps
-;; Version: 0.1.0
-;; Package-Requires: ((emacs "28.0"))
-;; Keywords: printer, transient, syntax-highlighting
+;; Version: 1.0.0
```

Comment:
@swflint is this correct? Requiring the current package in a file inside the package?

Reply:
Yes, because the way this will be published to MELPA is as three packages:
 - `transient-extras` as a common library
 - `transient-extras-lp` as one package
 - `transient-extras-a2ps` as another package.

This also means that the library can be used by other authors, and only certain parts can be installed as desired.